src/app/feed/feed-post/feed-post.component.ts

```diff
@@ -159,6 +171,46 @@ export class FeedPostComponent implements OnInit {
     });
   }
 
+  openDiamondsModal(event) {
+    event.stopPropagation();
+    this.modalService.show(DiamondsModalComponent, {
+      class: "modal-dialog-centered",
+      initialState: {
+        postHashHex: this.post.PostHashHex,
+      },
+    });
+  }
+
+  openLikesModal(event, imageURL) {
+    event.stopPropagation();
+    this.modalService.show(LikesModalComponent, {
+      class: "modal-dialog-centered",
+      initialState: {
+        postHashHex: this.post.PostHashHex,
+      },
+    });
+  }
+
+  openRecloutsModal(event, imageURL) {
+    event.stopPropagation();
+    this.modalService.show(RecloutsModalComponent, {
+      class: "modal-dialog-centered",
+      initialState: {
+        postHashHex: this.post.PostHashHex,
+      },
+    });
+  }
+
+  openQuoteRecloutsModal(event, imageURL) {
+    event.stopPropagation();
+    this.modalService.show(QuoteRecloutsModalComponent, {
+      class: "modal-dialog-centered",
+      initialState: {
+        postHashHex: this.post.PostHashHex,
+      },
+    });
+  }
```

Comment:
Doesn't look like we use `imageURL` in any of these.  Also seems like we can refactor since the only thing that changes is the component passed in. 

```
openInteractionModal(event, component): void {
    event.stopPropagation();
    this.modalService.show(component, {
      class: "modal-dialog-centered",
      initialState: {
        postHashHex: this.post.PostHashHex,
      },
    });
}
```

and then you could simplify each of these, for example:
```
openQuoteRecloutsModal(event): void {
    this.openInteractionModal(event, QuoteRecloutsModalComponent);
}
```